src/aquarium-optimized/dawn/ContextDawn.cpp

```diff
@@ -290,7 +291,7 @@ bool ContextDawn::GetHardwareAdapter(
     // Get an adapter for the backend to use, and create the Device.
     for (auto &adapter : instance->GetAdapters())
     {
-        if (adapter.GetBackendType() == backendType)
+        if (adapter.GetBackendType() == GetBackendType(backendType))

```

Comment:
@Jiawei-Shao The way to get backend from adapter returns dawn_native::BackendType, but creating dawn context is using wgpu::BackendType. Here are some reduntant code to compare the backend. Do you think we should fix it in Dawn?

Reply:
Because dawn_native::BackendType is deprecated, we should use GetProperties method of wgpu instead.